src/appsco-customers-page.html

```diff
@@ -0,0 +1,540 @@
+            :host .flex-vertical {
+                @apply(--layout-vertical);
+            }
+            :host .flex {
+                @apply(--layout-flex);
+            }
+            :host([tablet-screen]) {
+                --info-width: 250px;
+            }
+            :host([mobile-screen]) {
+                --info-width: 100%;
+            }
+            :host([screen992]) {
+                --account-basic-info: {
+                    width: 140px;
+                };
+                --account-basic-info-values: {
+                    width: 140px;
+                };
+
+                --account-auth-type-info: {
+                    width: 140px;
+                };
+                --account-auth-type-info-values: {
+                    width: 140px;
+                };
+            }
+        </style>
+
+        <iron-media-query query="(max-width: 992px)" query-matches="{{ screen992 }}"></iron-media-query>
+        <iron-media-query query="(max-width: 800px)" query-matches="{{ tabletScreen }}"></iron-media-query>
+        <iron-media-query query="(max-width: 600px)" query-matches="{{ mobileScreen }}"></iron-media-query>
+
+        <appsco-content id="appscoContent">
+
+            <div content>
+                <div class="customers-container">
+                    <appsco-company-customers id="appsoCustomers"
+                                           customers-api="[[ customersApi ]]"
+                                           authorization-token="[[ authorizationToken ]]"
+                                           size="100"
+                                           hide-total
+                                           selectable
+                                           on-loaded="_onCustomersLoaded"
+                                           on-empty-load="_onCustomersEmptyLoad"
+                                           on-customer-selected="_onCustomerselected"></appsco-company-customers>
+                </div>
+            </div>
+
+            <div class="flex-vertical" info>
+
+                <div class="info-header flex-horizontal">
+                    <appsco-company-customer-image customer="[[ customer ]]" preview></appsco-company-customer-image>
+                    <span class="customer-name flex">[[ customer.name ]]</span>
+                </div>
+
+                <div class="info-content flex-vertical">
+                    <appsco-company-customer-resources
+                            id="appscoCompanyCustomerResources"
+                            class="customer-items"
+                            authorization-token="[[ authorizationToken ]]"
+                            list-api="[[ _customerResourcesApi ]]"
+                            size="5"
+                            customer="[[ customer ]]"
+                            load-more
+                            preview
+                    ></appsco-company-customer-resources>
+
+                    <appsco-company-customer-roles
```

Comment:
Component appsco-company-customer-roles does not exist, and it appears that it is not needed. If I am not mistaken this entire div element (info section content) should stay empty for now?